packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -20,83 +20,109 @@
+            >
+              <ul class="uk-list">
+                <li>
+                  <file-upload
+                    :path="currentPath"
+                    :headers="headers"
+                    @success="onFileSuccess"
+                    @error="onFileError"
+                    @progress="onFileProgress"
+                  />
+                </li>
+                <li v-if="checkIfBrowserSupportsFolderUpload">
+                  <folder-upload
+                    v-if="!isIE11()"
+                    :root-path="currentPath"
+                    :path="currentPath"
+                    :headers="headers"
+                    @success="onFileSuccess"
+                    @error="onFileError"
+                    @progress="onFileProgress"
+                  />
+                </li>
+                <li>
+                  <div>
+                    <oc-button
+                      id="new-folder-btn"
+                      appearance="raw"
+                      class="uk-width-1-1"
+                      justify-content="left"
+                      @click="showCreateResourceModal"
+                    >
+                      <oc-icon name="create_new_folder" />
+                      <translate>New folder…</translate>
+                    </oc-button>
+                  </div>
+                </li>
+                <li v-for="(newFileHandler, key) in newFileHandlers" :key="key">
+                  <div>
+                    <oc-button
+                      appearance="raw"
+                      justify-content="left"
+                      :class="['new-file-btn-' + newFileHandler.ext, 'uk-width-1-1']"
+                      @click="
+                        showCreateResourceModal(false, newFileHandler.ext, newFileHandler.action)
+                      "
+                    >
+                      <oc-icon :name="newFileHandler.icon || 'save'" />
+                      <span>{{ newFileHandler.menuTitle($gettext) }}</span>
+                    </oc-button>
+                  </div>
+                </li>
+              </ul>
+            </oc-drop>
+          </template>
+          <size-info v-if="selectedFiles.length > 0" class="oc-mr-s uk-visible@l" />
+          <batch-actions />
+        </div>
+        <oc-button
+          id="files-view-options-btn"
+          key="files-view-options-btn"
+          v-oc-tooltip="newButtonTooltip"
+          data-testid="files-view-options-btn"
+          :aria-label="newButtonAriaLabel"
+          variation="passive"
+          appearance="outline"
+          size="small"
+        >
+          <oc-icon name="tune" size="small" />
+          <translate>View</translate>
+        </oc-button>
+        <oc-drop drop-id="files-view-options-drop" toggle="#files-view-options-btn" mode="click">
+          <oc-list>
+            <li>
+              <oc-switch
+                v-model="hiddenFilesShownModel"
+                data-testid="files-switch-hidden-files"
```

Comment:
```suggestion
                data-test-id="files-switch-hidden-files"
```
Sorry for being _this_ picky 🙈 just to keep it in sync

Reply:
https://github.com/owncloud/web/discussions/5367

Reply:
not decided, yet, so please keep it consistent (as `data-test-id`) or put pressure on the discussion. 😁

Reply:
Damn sry wasn't aware of the discussion, happy to go with either (and totally get the arguments to stay in synch with the vendor) as long as we're consequential 🧶